src/inmanta/agent/handler.py

```diff
@@ -83,6 +84,15 @@ class SkipResource(Exception):
     """
     A handler should raise this exception when a resource should be skipped. The resource will be marked as skipped
     instead of failed.
+    The resource will be skipped with a BlockedStatus of NO.
```

Comment:
I can't immediately think of any. It offers a broader range of cases where a handler developer could use it, but I'm also inclined to stick with your proposal. Mostly I think we have to make the decision (whatever it is) so that both sides know what to expect / provide.